tests/multiblocks_per_env/__init__.py

```diff
@@ -3,3 +3,4 @@
 # SPDX-License-Identifier: BSD-3-Clause
```

Comment:
Should we rename the top folder "mulitblocks_per_env" -> "mulitple_cuda_blocks_per_env"?